tests/reference/asr-test_bitwise_on_complex-dd9568b.stderr

```diff
@@ -1,5 +1,5 @@
-semantic error: Unsupported binary operation on complex: '|'
- --> tests/errors/test_bitwise_on_complex.py:8:11
+semantic error: Type mismatch in binary operator; the types must be compatible
+ --> tests/errors/test_bitwise_on_complex.py:5:10
   |
-8 |     print(c1 | c2)
-  |           ^^^^^^^ 
+5 |     c1 = 4+5j
+  |          ^ ^^ type mismatch (i32 and c64)
```

Comment:
different error

Reply:
Fixed in [b5b4c82](https://github.com/lcompilers/lpython/pull/1271/commits/b5b4c82342721669d793585c84d3c87b922f53ae)